### PR TITLE
Debrief page tuning

### LINKED ISSUE #223: Link to debrief page + datepicker
1. Add link to `/debrief/` page
2. Add Bootstrap's datepicker to that page

This should land after #221.

### PR DESCRIPTION
**Pull request by pbanaszkiewicz**
Resolves #223

### CODE REVIEW

File: workshops/views.py [hunk 1]
```diff
@@ -605,33 +605,34 @@ def search(request):
...
-    context = {'title' : 'Debrief',
-               'form' : form,
-               'all_tasks' : tasks}
+    context = {'title': 'Debrief',
+               'form': form,
+               'all_tasks': tasks}
     return render(request, 'workshops/debrief.html', context)
 
-#------------------------------------------------------------
+# ------------------------------------------------------------
```
Comment (wking) [inline workshops/views.py]: I'd just get rid of these markers ;).

**Comment by wking** [comment]
I haven't actually tested this, but the changes look good to me.  I
think we still want to use Bower instead of committing external code
to our repository (it makes ‘git grep …’ less useful, among other
things), but I still haven't gotten around to writing that up (see
earlier discussion in #105 and [1]).  Anyhow, I'm fine with this
landing whether or not you fix/drop the ‘#---…’ divider change ;).

[1]: https://github.com/swcarpentry/amy/pull/110#discussion_r22321871

**Comment by pbanaszkiewicz** [comment]
Hey @wking,

> I think we still want to use Bower instead of committing external code to our repository

Yes, definitely. I have zero experience with Bower, unfortunately. (I can use this issue as an excuse to gain some!)

> I'd just get rid of these markers ;).

Hm, I was thinking about splitting `views.py` into `views/` Python package with this content:

* `views/base.py` for `CreateViewContext`, `UpdateViewContext`, and `LoginRequiredMixin` classes
* `views/index.py` for `index` view
* `views/sites.py` for `all_sites`, `site_details`, `SiteCreate`, and `SiteUpdate` views
* similarily `views/airports.py`, `views/persons.py`, `views/events.py`, `views/tasks.py`, `views/badges.py`
* `views/tools.py` for `search`, `export`, `instructors`, and `debrief`.

If you agree then I'd:

1. rollback some changes I've made (excessive newlines, '#---' → '# ---')
2. merge this PR (I'd like someone to test it though)
3. open new issue for moving to Bower
4. open new issue for splitting `views.py`

**Comment by wking** [comment]
On Sat, Apr 04, 2015 at 12:44:36AM -0700, Piotr Banaszkiewicz wrote:
> > I think we still want to use Bower instead of committing external
> > code to our repository
> 
> Yes, definitely. I have zero experience with Bower,
> unfortunately. (I can use this issue as an excuse to gain some!)

It's pretty straightforward.  Install it [1], fill out a bower.json
with your dependencies [2], fill out a .bowerrc (also in amy's root
directory) with something like:

  $ cat >.bowerrc <<EOF
  {
    "directory": "workshops/static/bower_components/"
  }

and run ‘bower install’.  You may also want a user-wide ~/.bowerrc to
disable analytics [3].

> Hm, I was thinking about splitting `views.py` into `views/` Python
> package with this content…
> …
> If you agree then I'd:
> 
> 1. rollback some changes I've made (excessive newlines, '#---' → '# ---')
> 2. merge this PR (I'd like someone to test it though)
> 3. open new issue for moving to Bower
> 4. open new issue for splitting `views.py`

Sounds great.  I'll report back after I've tested this PR.

[1]: http://bower.io/#install-bower
[2]: http://bower.io/docs/creating-packages/#bowerjson
[3]: http://bower.io/docs/config/#analytics

**Comment by pbanaszkiewicz** [comment]
Bump.

**Comment by wking** [comment]
On Mon, Apr 13, 2015 at 01:38:07PM -0700, Piotr Banaszkiewicz wrote:
> Bump.

Thanks.  Tested locally and it looks good to me.

**Comment by pbanaszkiewicz** [comment]
Thanks!

**Pull request merged**